src/ray/gcs/gcs_client/test/gcs_client_test.cc

```diff
@@ -987,9 +1003,21 @@ TEST_P(GcsClientTest, TestEvictExpiredDestroyedActors) {
   }
 }
 
+TEST_P(GcsClientTest, TestGcsAuth) {
```

Comment:
Can you test some negative tests? Like the auth is set incorrectly and GCS just reject the call?

Reply:
That's tested in the next PR :)